packages/lib/src/vis/shared/AxisSystemContext.tsx

```diff
@@ -12,15 +12,15 @@ export interface AxisSystemParams {
   ordinateScale: AxisScale;
   dataToWorld: (vec: Vector2 | Vector3) => Vector2;
   worldToData: (vec: Vector2 | Vector3) => Vector2;
+
+  // For internal use only
```

Comment:
And this comment makes it clear to us that anything underneath is not documented and can be broken without worry :smile: